### DISCUSSION TITLE
SmartSeq STARsolo seg fault with readFilesManifest

### DISCUSSION THREAD

**Discussion by jerkolju** [alexdobin/STAR#1694] [Q&A]
Hi I'm facing this error while trying to map SmartSeq FastQ files 

EXITING because of FATAL ERROR: Genome version: 20201 is INCOMPATIBLE with running STAR version: 2.7.10b
SOLUTION: please re-generate genome from scratch with running version of STAR, or with version: 2.7.4a

Nov 02 19:34:39 ...... FATAL ERROR, exiting

I'm new to using other tools than cellRanger so I'm not sure what that error means

**Comment by alexdobin** [comment]
Hi @jerkolju 

you would need to re-generate the genome index with the STAR version that you are using. Please see the STAR manual which explains how to do it.

**Reply by jerkolju** [reply]
Thank you so much I managed to do the mapping !
